src/main/java/com/adobe/campaign/tests/integro/phased/PhasedTestManager.java

```diff
@@ -1756,14 +1753,16 @@ public void importFromString(String in_importString) {
             this.failedStep =
                 !l_valueArray[2].trim().isEmpty() ? l_valueArray[2] : NOT_APPLICABLE_STEP_NAME;
 
-            try {
-                this.setFailedInPhase(!l_valueArray[3].trim().isEmpty() ? Phases.valueOf(
-                    l_valueArray[3]) : Phases.NON_PHASED);
-            } catch (IllegalArgumentException exc) {
-                throw new IllegalArgumentException(
-                    "The given import string " + in_importString
-                        + " does not allow us to deduce the Phase.");
-            }
+                var importedPhase = !l_valueArray[3].trim().isEmpty() ?
+                        l_valueArray[3] : "N/A";
```

Comment:
i know that is an existing code, but we probably may to indicate the corresponding values to the l_valueArray[x] data.